app/services/authentication/providers/github.rb

```diff
@@ -25,10 +26,22 @@ def existing_user_data
         }
       end
 
+      def self.user_created_at_field
```

Comment:
Tbh I liked them better when they were instance methods, as you always seem to be calling them from an instance context as illustrated by the `provider.class.user_created_at_field` calls. Ruby is not quite as strict as Python about making things static when you don't access `self` in the method body. As a sort of compromise you can leave them as is and add 

```ruby
delegate :user_created_at_field, to: :class
```

Which will keep them on the class but make them a bit less awkward to acess from an instance.

Reply:
Awesome!